packages/ui/react-ui-form/src/components/deprecated/FormInput.tsx

```diff
@@ -49,7 +65,11 @@ export const DeprecatedFormInput = ({
     return (
       <div role='none' className='flex flex-col w-full'>
         {primary}
-        {secondary}
+        {secondary && (
+          <div role='none' className='max-w-[40rem]'>
```

Comment:
Does 40rem mean something here? What is the goal? Should this just apply `textBlockWidth`?